docs/deployment/index.rst

```diff
@@ -0,0 +1,266 @@
+   output the URL to FLINT Cloud (ingress).
+
+To teardown the infrastructure and delete the application, run
+``terraform destroy`` in the same directory as where ``main.tf`` is
+present. In case this fails, simply run it again.
+
+.. _Terraform: https://www.terraform.io/
+
+Azure
+-----
+
+To deploy FLINT.Cloud to Azure, we recommend using the Azure App Service
+using a custom Microsoft Container Registry container, built from our
+``local`` directory. This setup includes the usage of `Azure CLI`_ to
+deploy the infrastructure and the application to be used. You will also
+need to sign in to the Azure CLI by using the ``az login`` command. To
+finish the authentication process, follow the steps displayed in your
+terminal. To build the images, we would be using `Docker`_ and would be
+pushing them to `Azure Container Registry`_.
+
+Download the project
+~~~~~~~~~~~~~~~~~~~~
+
+Clone the repository:
+
+.. code:: sh
+
+   git clone https://github.com/moja-global/flint.cloud
+
+Navigate to the ``local`` directory:
+
+.. code:: sh
+
+   sh local
+
+Build the images locally
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To build the ``rest_api_gcbm`` image locally, run the following command:
+
+.. code:: sh
+
+   pushd rest_api_gcbm
+   docker build --build-arg BUILD_TYPE=RELEASE --build-arg NUM_CPU=4 -t gcbm-api .
+   popd
+
+To build the ``rest_api_flint.example`` image locally, run the following
+command:
+
+.. code:: sh
+
+   pushd rest_api_flint.example
+   docker build -t flint-api .
+   popd
+
+Create a resource group
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now prepare resources in Azure to which you can push the image
+and then deploy a container to Azure App Service. You can start by
+creating a resource group in which you want to collect all the
+resources.
+
+.. code:: sh
+
+   az group create --name myResourceGroup --location centralindia
+
+You can change the ``--location`` value to specify a region near you.
+
+Create a Container Registry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now push the image to Azure Container Registry from which App
+Service can deploy it. Create an Azure Container Registry to push your
+images to:
+
+.. code:: sh
+
+   az acr create --name <registry-name> --resource-group myResourceGroup --sku Basic --admin-enabled true
+
+Replace ``<registry-name>`` with a suitable name for your registry. The
+name must contain only letters, numbers, and must be unique across all
+of Azure.
+
+Retrieve your credentials for the Container Registry:
+
+.. code:: sh
+
+   az acr credential show --resource-group myResourceGroup --name <registry-name>
+
+Use the ``docker login`` command to sign in to the container registry:
+
+.. code:: sh
+
+   docker login <registry-name>.azurecr.io --username <registry-username>
+
+Replace ``<registry-name>`` and ``<registry-username>`` with values from
+the previous steps. When prompted, type in one of the passwords from the
+previous step.
+
+Let's tag the images with the registry name:
```

Comment:
```suggestion
Tag the images with the registry name:
```
Google's style guide recommends to avoid this kind of phrase: https://developers.google.com/style/tone?hl=en#some-things-to-avoid-where-possible (bullet 12).